kbfshash/hash_test.go

```diff
@@ -226,3 +226,24 @@ func TestVerify(t *testing.T) {
 	err = corruptHMAC.Verify(key, data)
 	require.IsType(t, HashMismatchError{}, errors.Cause(err))
 }
+
+var defaultHashBenchmarkResult bool
+var defaultHashBenchmarkError error
+
+func BenchmarkDefaultHash(b *testing.B) {
+	data := make([]byte, b.N)
+	for i := 0; i < b.N; i++ {
```

Comment:
It would be nicer to use a passed in param for the length of the data (and rename it), and then let `b.N` only determine the number of runs, which I think is the intention.  Then you could have several benchmarks with different hardcoded data lengths that call this common function..